src/DonateHope.Core/ServiceContracts/CampaignsServiceContracts/ICampaignRetrieveService.cs

```diff
@@ -10,4 +10,5 @@ public interface ICampaignRetrieveService
     Task<Result<IEnumerable<CampaignGetResponseDto>>> GetCampaigns();
     Task<Result<CampaignGetResponseDto>> GetCampaignByIdAsync(Guid campaignId);
     Task<Result<IEnumerable<CampaignGetResponseDto>>> FilterCampaigns(string keyword);
+    Task<Result<IEnumerable<CampaignGetResponseDto>>> GetTop3HighestRatingCampaigns();
```

Comment:
We not too specific here - I think top highest should be enough. The remaining should be in the inner logic of the method

Reply:
updated